### ISSUE TITLE
Links in News Listing widget should read link text instead of alt text (Level A failure)

### ISSUE THREAD

**Issue by mmafe** [ccswbs/ugnext#364]
Similar to #358 

<img width="1579" height="754" alt="Image" src="https://github.com/user-attachments/assets/6643dbab-59ab-4e79-8c4b-0abb35f10640" />

**Comment by mmafe** [comment]
Fixed
